packages/@vuepress/theme-default/layouts/Layout.vue

```diff
@@ -120,6 +120,7 @@ export default {
 
   methods: {
     toggleSidebar (to) {
+      this.$emit("toggle-sidebar", this.isSidebarOpen)
```

Comment:
This should be placed after `this.isSidebarOpen` gets its new value. Otherwise you'll emit the wrong state.

Reply:
Good catch, updated!